### PR TITLE
Feature 35 integrate database storage with analysis script

### PR DESCRIPTION
**Pull request by ValantineSuh**
implemented functionality to connect database to analyzed  data

### CODE REVIEW

**Comment by ValantineSuh** [comment]
hello @AssahBismarkabah @stephane-segning  i created a pull request on relating the analyzed data to the database

**Comment by AssahBismarkabah** [comment]
ok then we should just wait for sirs reviews and approvals

File: app.py
Comment (stephane-segning) [inline app.py:6]: What is this secret for?

File: scraping.py
Comment (stephane-segning) [inline scraping.py:1]: Why change this file? Is it related to your code?

**Review by stephane-segning — CHANGES_REQUESTED** [review]
I cannot tell exactly what your changes are. So please explain it to me in the next meeting

File: app.py
Comment (ValantineSuh) [reply app.py:6]: > What is this secret for?
Greetings @stephane-segning 
Please Sir, concerning the above secret key, I've been trying to figure it out.
Wish to inform you that I and my team did not add any key to the app.py

File: scraping.py
Comment (ValantineSuh) [reply scraping.py:1]: > Why change this file? Is it related to your code?
Sir we later discovered that you asked the other teams to remove the scraping parameter, which was based on the images and the text. so we had to edit the scraping functionality without specifying the specific data to extract. as seen in the function below
`def scrape_website(url, depth):`

File: app.py
Comment (stephane-segning) [reply app.py:6]: Then remove it na

File: scraping.py
Comment (stephane-segning) [reply scraping.py:1]: So, this code should then change

**Review by stephane-segning — CHANGES_REQUESTED** [review]

**Comment by ValantineSuh** [comment]
Ok Sir @stephane-segning 
We are working on it

**Comment by ValantineSuh** [comment]
hello sir @stephane-segning   we are done making the necessary changes in the app.py by importing the functions required for the database functionality

**Comment by AssahBismarkabah** [comment]
we also removed the 

```
app = Flask(__name__)
app.secret_key = 'your_secret_key'  # 

```
which was not part of our   work

**Pull request closed without merging**
